cvmfs/catalog_mgr_rw.cc

```diff
@@ -12,9 +12,12 @@
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
+#include <ctime>
```

Comment:
As I said, a separate pull request for necessary fixes which will be merged first.  The new feature should then be a patch on top of the clean(er) code base.